vectordatasource/transform.py

```diff
@@ -4267,3 +4268,76 @@ def truncate_min_zoom_to_2dp(shape, properties, fid, zoom):
         properties['min_zoom'] = round(min_zoom, 2)
 
     return shape, properties, fid
+
+
+class Palette(object):
```

Comment:
Sure, it's nice to give a hat-tip. Although the idea of palettizing to a small, fixed set of colours goes way back before that. Fixed in 9657644.